os/binfmt/binfmt_exit.c

```diff
@@ -114,6 +118,19 @@ int binfmt_exit(FAR struct binary_s *bin)
 		berr("ERROR: unload_module() failed: %d\n", ret);
 	}
 
+#ifdef CONFIG_SAVE_BIN_SECTION_ADDR
```

Comment:
1. same as save function, let's make this as a new function.
2. How about making a new file to get this and save?
  Because they are not matched with file name,

Reply:
Modified. Thanks.